accounts/tokens.py

```diff
@@ -0,0 +1,19 @@
+"""
+Token generator module for email verification
+"""
+from django.contrib.auth.tokens import PasswordResetTokenGenerator
+from django.utils import six
+
+
+class TokenGenerator(PasswordResetTokenGenerator):
+    """
+    Token generator for email verification
+    """
+    def _make_hash_value(self, user, timestamp):
```

Comment:
Is there any specific reason for overriding this method? Normally `private` methods are meant to be not modified or accessed. They are part of the internal implementation.

Reply:
no specific reason , tested it and it works just fine without overriding internal method.This piece of code is removed and pushed into repo.